pkg/querier/http.go

```diff
@@ -73,6 +73,7 @@ func (q *QuerierAPI) RangeQueryHandler(ctx context.Context, req *queryrange.Loki
 		return logqlmodel.Result{}, err
 	}
 
+	// TODO: fun fact req should implement params. So no wrapper would be required 🤷
```

Comment:
The `logql.Params` interface is defined as
```go
type Params interface {
	Query() string
	Start() time.Time
	End() time.Time
	Step() time.Duration
	Interval() time.Duration
	Limit() uint32
	Direction() logproto.Direction
	Shards() []string
}
```

`queryrange.LokiRequest` is very similar but has `GetEndTs` instead of `End`. I'd not change it here but wanted to surface the difference.